pages/common/gh-gist.md

```diff
@@ -3,22 +3,22 @@
 > Work with GitHub Gists on the command-line.
 > More information: <https://cli.github.com/manual/gh_gist>.
 
-- Create a new Gist from a space-separated list of files:
+- Create a new Gist from specific files:
 
-`gh gist create {{path/to/files}}`
+`gh gist create {{path/to/file1 path/to/file2 ...(1..)}}`
```

Comment:
This syntax is not intuitive.

I do not see why this has to change. If there has to be a change for the sake of it, this might be more intuitive to read:
```suggestion
`gh gist create {{path/to/file1}} {{path/to/file2}} ...}}`
```

Reply:
I agree. :) I've already declined the idea to extend ellipsis in a such way.

Reply:
`{{path/to/file1}} {{path/to/file2 path/to/file3 ...}}` is used as ellipsis in incorrect with just one item and `...`.